config/legacyconfig/config.go

```diff
@@ -123,6 +175,7 @@ type BaseHelm struct {
 	Init               InitValues       `json:"init,omitempty"`
 	MultiNamespaceMode EnabledSwitch    `json:"multiNamespaceMode,omitempty"`
 	Telemetry          TelemetryValues  `json:"telemetry,omitempty"`
+	ServiceCIDR        string           `json:"serviceCIDR,omitempty"`
```

Comment:
Had to add this in order for 0.19 configs (where `serviceCIDR` is set per default) to work with the strict decoding with `DisallowUnknownFields`.

Reply:
You could implement a custom [Unmarshaller](https://pkg.go.dev/encoding/json#Unmarshaler), that handles this specific case of unmarshalling, while leaving the rest to a DisallowUnknownFields.

Reply:
Good hint! But I would prefer to actually align the actual allowed fields in legacyconfig before doing exceptions/implement a custom marshaller. But if there is a good reason to **not** set this field here, I could totally go this route 🙂

Reply:
Probably @FabianKramm has more context?!

Reply:
This field was just forgotten here, so your solution @johannesfrey is correct, I wouldn't change that